File: C++/A/681A-agoodcontest.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin >> n;
    bool perform_good = 0;
    while (n--)
    {
        string handle;
        int before, after;
        cin >> handle >> before >> after;
        if (before >= 2400 && after > before)
        {
            perform_good = true;
            break;
        }
    }
    cout << ((perform_good) ? "YES" : "NO");
}

File: Python/A/681A-agoodcontest.py
n = int(input())
perform_good = False
for _ in range(n):
    handle, before, after = input().split()

    before_rating, after_rating = int(before), int(after)

    if before_rating >= 2400 and after_rating > before_rating:
        perform_good = True
        break
print("YES" if perform_good else "NO")
